apps/daimo-mobile/src/view/screen/InviteScreen.tsx

```diff
@@ -127,18 +137,25 @@ function HeaderCountText({
   invitees?: EAccount[];
   usesLeft?: number;
 }) {
+  const nav = useNav();
+
   const showInviteesCount = invitees != null && invitees.length > 0;
   const showUsesLeft = usesLeft != null && (showInviteesCount || usesLeft > 0);
 
   return (
     <View>
       {showInviteesCount && (
-        <TextCenter>
-          <TextBody color={color.primary}>
-            You've invited {invitees?.length}{" "}
-            {invitees?.length === 1 ? "friend" : "friends"}
-          </TextBody>
-        </TextCenter>
+        <Pressable
+          onPress={() => nav.push("YourInvites")}
+          children={({ pressed }) => (
```

Comment:
same here you can try hitSlip 16 or 32 to see what feels right

Reply:
16 feels good to me, done